src/site/content/en/blog/badging-api/index.md

```diff
@@ -158,9 +153,52 @@ Android only ever shows a dot instead of a numeric value.
 Don't assume anything about how the user agent wants to display the badge.
 Some user agents may take a number like "4000" and rewrite it as
 "99+". If you saturate the badge yourself (for example by setting it to "99")
-then the "+" won't appear. No matter the actual number, just set
-`Badge.set(unreadCount)` and let the user agent deal with displaying it
-accordingly.
+then the "+" won't appear. No matter the actual number, just call
+`setExperimentalAppBadge(unreadCount)` and let the user agent deal with
+displaying it accordingly.
+
+While the Badging API *in Chrome* requires an installed app with an icon
+that can be badged, you shouldn't make calls to the Badging API dependent
+on the install state. Just call the API when it exists. If it works,
```

Comment:
'you shouldn't make calls to the Badging API dependent on the install state'

Is that because install states aren't comprehensible or meaningful to non-technical users? Maybe you should say why this is bad.

Reply:
No, badges won't show up in Chrome unless the app is installed, as we need a dock/launcher icon to display them on. Setting a badge in an app that's in a tab is a no op in Chrome but it **might** do something in another browser (or in Chrome, in the future).

I think this is less important with the current version of the API than the old one, as we now have separate setClientBadge and setAppBadge methods.

Reply:
I should have asked this differently. I'm still not clear on why I shouldn't make calls to the API depend on install state.

Reply:
The user agent may choose to do something when the app is not installed. You don't want to break this.

Reply:
Can you please make that clear in the article.

Reply:
Done.